examples/kubernetes/README.rst

```diff
@@ -0,0 +1,58 @@
+Kubernetes Deployment
+=====================
+
+This directory contains all Cilium deployment files that can be used in
+Kubernetes.
+
+Each directory represents a Kubernetes version, from :code:`1.7` to :code:`1.11`,
+and inside each version there is a list of files to deploy Cilium.
+
+The structure directory will be :code:`${k8s_major_version}.${k8s_minor_version}/*.yaml`.
+
+To generate those files simply run :code:`make`, which will pick up the Cilium
+version from the :code:`VERSION` file at the root of the repo, or if you want to
+specify the Cilium version yourself use :code:`make CILIUM_VERSION=X.Y.Z`.
+
+If you want to clean up a specific version, run :code:`make clean` which will
+delete all generated files.
+
```

Comment:
If you decide to go ahead with `cleanall` or `veryclean` add comment about all directories being cleanup/removed up as well.